src/test/java/guitests/ClearCommandTest.java

```diff
@@ -0,0 +1,35 @@
+package guitests;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+import seedu.toluist.model.Task;
+
+/**
+ * Gui tests for clear command
+ */
+public class ClearCommandTest extends ToLuistGuiTest {
+
+    @Test
+    public void clearCurrentTasks() {
+        String command = "clear";
+        commandBox.runCommand(command);
+        assertTrue(getTasksShown().isEmpty());
+
+    }
+
+    @Test
+    public void addAndRemoveATask() {
+        String taskDescription = "drink Koi after school";
```

Comment:
Using add command here is fine since gui tests are end-to-end testing, not unit test